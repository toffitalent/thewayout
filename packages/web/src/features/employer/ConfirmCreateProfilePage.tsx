import { Button, Flex, Text } from '@disruptive-labs/ui';
import { useRouter } from 'next/router';
import { useCallback } from 'react';
import { SEO } from '@app/components/SEO';

export const ConfirmCreateProfilePage = () => {
  const router = useRouter();

  const handleClick = useCallback(() => router.replace('/employer/'), [router]);

  return (
    <>
      <SEO title="Profile Created" />
      <Flex maxWidth={96} ml="auto" mr="auto">
        <Text fontSize="2xl" fontWeight="700">
          Employer added!
        </Text>
        <Button colorScheme="primary" mt={10} fluid onClick={handleClick}>
          Done!
        </Button>
      </Flex>
    </>
  );
};
