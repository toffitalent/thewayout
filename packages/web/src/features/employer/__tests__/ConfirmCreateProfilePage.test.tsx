import { render } from '@test';
import { ConfirmCreateProfilePage } from '../ConfirmCreateProfilePage';

describe('ConfirmCreateProfilePage', () => {
  test('renders correctly', () => {
    const { container } = render(<ConfirmCreateProfilePage />);
    expect(container).toMatchSnapshot();
  });
});
