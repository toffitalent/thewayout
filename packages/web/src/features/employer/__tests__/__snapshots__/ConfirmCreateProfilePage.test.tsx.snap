// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmCreateProfilePage renders correctly 1`] = `
<div>
  <div
    class="maxWidth-96 marginRight-auto marginLeft-auto"
  >
    <p
      class="text fontSize-2xl fontWeight-700"
    >
      Employer added!
    </p>
    <button
      class="button md solid c-primary fluid marginTop-10"
      type="button"
    >
      Done!
    </button>
  </div>
</div>
`;
